algorithms/kernel/dtrees/dtrees_feature_type_helper.i

```diff
@@ -205,7 +205,7 @@ template <typename IndexType, typename algorithmFPType, CpuType cpu>
 services::Status ColIndexTaskBins<IndexType, algorithmFPType, cpu>::makeIndex(NumericTable& nt,
     IndexedFeatures::FeatureEntry& entry, IndexType* aRes, size_t iCol, size_t nRows, bool bUnorderedFeature)
 {
-    if(bUnorderedFeature || nRows <= _prm.maxBins*_prm.minBinSize)
```

Comment:
how it affected decision forest algorithm performance?

Reply:
It doesn't affect decision forest, in this algorithm "ColIndexTask" is used instead of "ColIndexTaskBins"